workflows/offline_ml_diags/offline_ml_diags/compute_diags.py

```diff
@@ -316,8 +288,9 @@ def main(args):
 
     logger.info("Opening ML model")
     model = fv3fit.load(args.model_path)
+    if "dQ1" in model.output_variables or "dQ2" in model.output_variables:
+        model = _derived_heating_moistening_model(model)
```

Comment:
Since you check the existence in and outside of the function, you could instead use a helper (e.g., `_check_for_derivable_outputs`) that returns a list of variable names `Q1` and/or `Q2` or  an empty list. Then if that list isn't empty, adjust the model into `fv3fit.DerivedModel`.

Reply:
Good idea, it's cleaner now and easy to extend if more derived outputs need to get added in the future.